package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --port 4600",
```

Comment:
You can specify port by default inside angular.json
`
"serve": {
  ...
  "options": {
    ...
    "port": 4600,
  },
}
`

Reply:
Thanks. Forgot about it, will do it in next PR 🙂